packages/firebase_auth/firebase_auth_platform_interface/lib/src/providers/oauth.dart

```diff
@@ -15,7 +15,7 @@ class OAuthProvider extends AuthProvider {
   OAuthProvider(String providerId) : super(providerId);
 
   List<String> _scopes = [];
-  Map<dynamic, dynamic>? _parameters;
+  Map<dynamic, dynamic> _parameters = {};
```

Comment:
@Salakar Can `parameters` be mutated (parameters['foo'] = 42)? I don't see a mutation in the codebase, but maybe I'm missing something.

Am I correct in thinking that users should use `setCustomParameters` instead?

If so, we can make this `{}` const

Reply:
I do not believe that you've missed anything so this can be made const. Other providers are using the same `parameters` map without a const map though. E.g., https://github.com/FirebaseExtended/flutterfire/blob/master/packages/firebase_auth/firebase_auth_platform_interface/lib/src/providers/twitter_auth.dart#L52